.travis.yml

```diff
@@ -41,6 +41,37 @@ before_install:
   - export GOFLAGS=-mod=vendor
   - if [ $TRAVIS_OS_NAME = "linux" ]; then sudo apt-get -qq update; sudo apt-get install -y dwz; echo "dwz version $(dwz --version)"; fi
   - if [ $TRAVIS_OS_NAME = "windows" ]; then choco install procdump make; fi
+
+
+# 386 linux
+jobs:
+  include:
+    -  os: linux
+       services: docker
+       env: go_32_version=1.14
+    -  os: linux
+       services: docker
+       env: go_32_version=1.13
```

Comment:
We're only going to test the latest version of go (1.14) on anything that isn't linux/amd64 as a compromise between coverage and speed.